19_js-forms/js-forms_handle-form-submit/js/index.js

```diff
@@ -1,3 +1,23 @@
 console.clear();
 
 const form = document.querySelector('[data-js="form"]');
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const formData = new FormData(event.target);
+  const data = Object.fromEntries(formData);
+
+  console.log("Object_", data);
+
+  const ageBadnessSum = event.target.badness.value + event.target.age.value;
```

Comment:
Is the `ageBadnessSum` calculation correct here? 

It seems like the values of `badness` and `age` are being concatenated as strings instead of being added as numbers.

Should these values be converted to numbers before summing them, e.g., using `parseInt` or `Number`? 🤔